packages/breadcrumb/README.md

```diff
@@ -0,0 +1,123 @@
+# @undercoatcsss/breadcrumb
+
+Breadcrumb is a component of the [Undercoat.css](https://github.com/undercoat/undercoat) framework.
+
+![Travis (.org)](https://img.shields.io/travis/undercoat/undercoat.svg?style=flat-square) ![npm (scoped)](https://img.shields.io/npm/v/@undercoatcss/breadcrumb?style=flat-square) ![WCAG 2.0 Level AA](https://img.shields.io/badge/WCAG_2.0-Level_AA-brightgreen.svg?style=flat-square) ![schema.org](https://img.shields.io/badge/microdata-schema.org-brightgreen.svg?style=flat-square)
+
+> **CAUTION! IT'S UNDER HEAVY DEVELOPMENT CURRENTLY! DON'T USE IT IN PRODUCTION!**
+
+## Description
+
+Breadcrumb is a logically and functionally independent reusable **page component** (block), which represents a **secondary navigation scheme** that reveals the user's location in a website.
+
+## Installation
+
+### NPM
+
+```bash
+npm install @undercoatcss/breadcrumb
+```
+
+## Usage
+
+After installation, you can link or import the assets into your project:
+
+### CDN
+
+```html
+<link rel="stylesheet" href="https://unpkg.com/@undercoatcss/breadcrumb@latest/dist/breadcrumb.min.css">
+```
+
+### Webpack
+
+```javascript
+import '@undercoatcss/breadcrumb/breadcrumb.css';
+```
+
+## Settings
+
+The component provides CSS custom property to change its defaults.
+
+| Property                               | Default value
+| -                                      | -
+| --breadcrumb--separator                | \2192
+
+## HTML markup
+
+### Without microdata
+
+```html
+<nav class="breadcrumb" aria-label="Breadcrumb">
+  <ul class="breadcrumb__list" role="list">
+    <li class="breadcrumb__item" role="listitem">
+      <a class="breadcrumb__link" href="#">Index</a>
+    </li>
+    <li class="breadcrumb__item" role="listitem">
+      <a class="breadcrumb__link" href="#">Category</a>
+    </li>
+    <li class="breadcrumb__item" role="listitem">
+      <a class="breadcrumb__link" aria-current="page">Subategory</a>
+    </li>
+  </ul>
+</nav>
+```
+
+### With microdata
+
+```html
+<nav class="breadcrumb" aria-label="Breadcrumb">
+  <ul class="breadcrumb__list" role="list" itemscope itemtype="http://schema.org/BreadcrumbList">
+    <li class="breadcrumb__item" role="listitem" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+      <span itemprop="name">
+        <a class="breadcrumbs__link" href="#" itemtype="http://schema.org/Thing" itemprop="item">Index</a>
```

Comment:
wrong class `breadcrumbs__link`
Should be `breadcrumb__link`

Reply:
Fixed in 974ba475647f219bfce5520a56b77e3a5b92941d.